api_practice/schemas.py

```diff
@@ -0,0 +1,34 @@
+from pydantic import BaseModel
```

Comment:
this entire class must be split into multiple class. Each class here should go to a separate file. 
Please make a package called models.. and put everything there. 
Normally python allows multiple classes in a single file. But let us keep one per file.

Reply:
Models directory created and the model file is created